Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/qrcode/consent/SubmissionConsentFragment.kt

```diff
@@ -0,0 +1,47 @@
+package de.rki.coronawarnapp.ui.submission.qrcode.consent
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.FragmentSubmissionConsentBinding
+import de.rki.coronawarnapp.ui.submission.viewmodel.SubmissionNavigationEvents
+import de.rki.coronawarnapp.util.di.AutoInject
+import de.rki.coronawarnapp.util.ui.doNavigate
+import de.rki.coronawarnapp.util.ui.observe2
+import de.rki.coronawarnapp.util.ui.viewBindingLazy
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModelFactoryProvider
+import de.rki.coronawarnapp.util.viewmodel.cwaViewModels
+import javax.inject.Inject
+
+class SubmissionConsentFragment : Fragment(R.layout.fragment_submission_consent), AutoInject {
+
+    @Inject lateinit var viewModelFactory: CWAViewModelFactoryProvider.Factory
+    private val viewModel: SubmissionConsentViewModel by cwaViewModels { viewModelFactory }
+    private val binding: FragmentSubmissionConsentBinding by viewBindingLazy()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding.viewModel = viewModel
+        binding.lifecycleOwner = this
```

Comment:
_binding.lifecycleOwner = this_ should be replaced with this.viewLifecycleOwner 
(can lead to fragment issues e.g. while navigating)

It's also little bit redundant because you already use viewBindingLazy above.
So to avoid this bug either adjust the lifecycleOwner line or simply remove it :snowman:

Reply:
I removed it. I missed that it is already applied.